benchmarks/cluster3nodes/configuration/log-config-rt-view-2.yaml

```diff
@@ -98,7 +98,7 @@ traceForwardTo:
 
 NodeId:
 Protocol: RealPBFT
-GenesisFile: configuration/genesis/latest-genesis
+GenesisFile: some/path
```

Comment:
Is this intended?

Reply:
yes, configuration is being overwritten by the start script with the absolute path to the genesis file. also a new requirement.